client/components/QuestionsAndAnswers/modal/QuestionForm.jsx

```diff
@@ -37,7 +37,7 @@ const QuestionForm = ({
           type="text"
           maxlength="60"
           placeholder="Example: jackson11!"
-          onChange={(e) => setNewQuestion({ ...newQuestion, asker_name: e.target.value })}
+          onChange={(e) => setNewQuestion({ ...newQuestion, name: e.target.value })}
         />
         <Notes>
           For privacy reasons, do not use your full name or email address
```

Comment:
Note about your handle submit - there isn't any validation yet to make sure all the fields have valid inputs / there are inputs before submitting to the server.